static/src/javascripts/projects/common/modules/experiments/tests/contributions-embed.js

```diff
@@ -0,0 +1,124 @@
+    'common/utils/$',
+    'common/utils/template',
+    'common/views/svg',
+    'common/utils/fastdom-promise',
+    'common/utils/mediator',
+    'text!common/views/contributions-embed.html',
+    'common/utils/robust',
+    'text!common/views/giraffe-message.html',
+    'inlineSvg!svgs/icon/arrow-right'
+
+], function (bean,
+             qwery,
+             $,
+             template,
+             svg,
+             fastdom,
+             mediator,
+             contributionsEmbed,
+             robust,
+             giraffeMessage,
+             arrowRight
+) {
+
+
+    return function () {
+
+        this.id = 'ContributionsEmbed20160823';
+        this.start = '2016-08-23';
+        this.expiry = '2016-08-30';
+        this.author = 'Jonathan Rankin';
+        this.description = 'Test contributions embed with amount picker.';
+        this.showForSensitive = false;
+        this.audience = 0.10;
+        this.audienceOffset = 0.13;
+        this.successMeasure = 'Impressions to number of contributions';
+        this.audienceCriteria = 'All users';
+        this.dataLinkNames = '';
+        this.idealOutcome = 'The interactive embed performs 3x better than the control';
+        this.canRun = function () {
+            var pageObj = window.guardian.config.page;
+            return !(pageObj.isSensitive || pageObj.isLiveBlog || pageObj.isFront || pageObj.isAdvertisementFeature) && pageObj.edition === 'UK';
+        };
+
+
+        var writer = function (component) {
+
+            return fastdom.write(function () {
+                var a = $('.submeta');
+                component.insertBefore(a);
+                mediator.emit('contributions-embed:insert');
+            });
+
+        };
+
+        var interactiveCompleter = function (complete) {
+            mediator.on('contributions-embed:insert', function () {
+                qwery('figure.interactive.contribute-embed').forEach(function (el) {
+                    require([el.getAttribute('data-interactive')], function (interactive) {
+                        robust.catchErrorsAndLog('interactive-bootstrap', function () {
+                            interactive.boot(el, document, window.guardian.config, mediator);
+                        });
+                    });
+
+                    require(['ophan/ng'], function(ophan) {
+                        var a = el.querySelector('a');
+                        var href = a && a.href;
+
+                        if (href) {
+                            ophan.trackComponentAttention(href, el);
+                        }
+                    });
+                });
+
+                bean.on(qwery('#giraffe__contribute-button')[0], 'click', function (){
+                    complete();
+                });
+
+            });
+        };
+
+        var controlCompleter = function (complete) {
+            mediator.on('giraffe:insert', function () {
+                bean.on(qwery('#giraffe__contribute')[0], 'click', function (){
```

Comment:
Just checking your click target here is `#giraffe__contribute` instead of button as above?